src/Forms/Validation/RequiredFieldsValidator.php

```diff
@@ -1,9 +1,12 @@
 <?php
 
-namespace SilverStripe\Forms;
+namespace SilverStripe\Forms\Validation;
 
 use SilverStripe\Core\ArrayLib;
 use SilverStripe\Dev\Deprecation;
+use SilverStripe\Forms\HasOneRelationFieldInterface;
```

Comment:
Arguably `HasOneRelationFieldInterface` should move into the new validation namespace since that's all it's used for - but I won't block merging if it's left where it is.

Reply:
I don't think it should move, while it is only used for validation, the interface itself has nothing to do with validation